### ISSUE TITLE
Toml selector .p2p not working

### ISSUE THREAD

**Issue by true-eye** [TomWright/dasel#44]
**Describe the bug**

I have a toml file which contains the following
```
[rpc]

abc = ""

[p2p]

persisent_peers = ""
```

I would like to select `p2p` but `-p toml .p2p` is not working
`-p toml .rpc` works. So I guess if it includes the number, then I can't use the selector?

**To Reproduce**
1. Go to https://dasel.tomwright.me/s/1faea436-e46e-49c0-bb44-6b30f9ed4a3c
2. Click on `Run`
3. See error

Include (sanitized) input files and commands to help along the way.

**Expected behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/46362478/99295113-1ad1da80-2899-11eb-9378-b253de207390.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS
 - Version [e.g. 22] (`dasel --version`) latest

**Additional context**

**Comment by TomWright** [comment]
Hey @true-eye,

It looks like this is an issue with the argument parsing on the playground rather than an issue with dasel itself.

I took the file content from the playground snippet and it works locally with dasel `v1.5.1`.

```
$ dasel -f tmp.toml .p2p
addr_book_file = "config/addrbook.json"
addr_book_strict = false
allow_duplicate_ip = false
dial_timeout = "3s"
external_address = ""
flush_throttle_timeout = "100ms"
handshake_timeout = "20s"
laddr = "tcp://0.0.0.0:26656"
max_num_inbound_peers = 40
max_num_outbound_peers = 10
max_packet_msg_payload_size = 1024
persistent_peers = ""
persistent_peers_max_dial_period = "0s"
pex = false
private_peer_ids = ""
recv_rate = 5120000
seed_mode = false
seeds = ""
send_rate = 5120000
unconditional_peer_ids = ""
upnp = false
```

As an aside, it's quite hard to parse the args on the playground because of the security requirements.

I believe this issue can be closed now but let me know if you're still having issues!

**Comment by TomWright** [comment]
Hey @true-eye,

The issue on the playground is now resolved. Just a note in-case you were waiting on it 🙂

**Comment by true-eye** [comment]
Cool!
Too fast. :) Thanks
